src/api/app/views/webui2/webui/staging/workflows/_problems.html.haml

```diff
@@ -1,21 +1,29 @@
 :ruby
-  max_problems = 5
+  max_shown = 5
   project_class = staging_project.name.tr(':', '_')
   item_project_class = 'hidden-item-' + project_class
+  checks_problems = staging_project.failed_status_checks
+  build_problems = staging_project.problems
+  total_problems = checks_problems.length + build_problems.length
+  rendered_checks_problems = [max_shown, checks_problems.length].min
+  rendered_build_problems = max_shown - rendered_checks_problems
```

Comment:
@ana06 i know its better to assign them after the else statement but rubocop was complaining :(